services/commands-service.ts

```diff
@@ -239,7 +239,11 @@ export class CommandsService implements ICommandsService {
 				}
 
 				if(data.words === 1) {
-					return tabtab.log(this.allCommands(false), data);
+					var allCommands = this.allCommands(false);
```

Comment:
Can we change false to consts / enum? In TypeScript 1.4, there is 'const enum' which is quite less painful than the full enum.